client/src/app/tabs/cmmn/CmmnEditor.js

```diff
@@ -122,6 +126,23 @@ export class CmmnEditor extends CachedComponent {
     onError(error);
   }
 
+  handleImport = (error, warnings) => {
+
+    const {
+      onImport
+    } = this.props;
+
+    if (error) {
```

Comment:
Why not simpler: `onImport(error, warnings)`?

Reply:
👍